ci_framework/roles/rhol_crc/tasks/add_crc_creds.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Make sure kubeconfig file exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_rhol_crc_kubeconfig }}"
+  register: crc_kubeconfig
+
+- name: Add crc kubeconfig to user's bashrc
+  when: crc_kubeconfig.stat.exists
+  block:
+    - name: Add crc creds in the user bashrc file
+      ansible.builtin.blockinfile:
+        dest: ~/.bashrc
```

Comment:
btw, what if we're not using bash? :/

Reply:
good point, if the file does not found, it will be created. Update is coming in next patchset.